curriculum/challenges/english/03-front-end-libraries/react/create-a-controlled-form.md

```diff
@@ -98,6 +98,17 @@ Submitting the form should run `handleSubmit` which should set the `submit` prop
 })();
 ```
 
+`handleSubmit` should call `event.preventDefault`
+
+```js
+assert(
+  MyForm.prototype.handleSubmit.toString()
+    .match(/(?<!\/\*\s*?)(?<!\/\/ *?)\bevent\.preventDefault\(\s*?\)/gs)
+    // First negative lookbehind checks for block comment start /*
+    // Second negative lookbehind checks for line comment start //
```

Comment:
I'm afraid we can't use lookbehinds in our tests, because Safari does not support them.  Is it possible to test this without them?

Reply:
I am wondering if we are able to write a linting rule with either our Eslint or CodeQL to catch and fail the CI for these?

Reply:
> I'm afraid we can't use lookbehinds in our tests, because Safari does not support them. Is it possible to test this without them?

That's unfortunate. But yes, the two negative lookbehinds can be replaced with two negative matches.